modules/__tests__/Router-test.js

```diff
@@ -272,7 +268,7 @@ describe('Router', function () {
         render() {
           const { label, children } = this.props
           const child = React.cloneElement(children, {
-            createElement: this.createElement
+            createElement: this.createElement.bind(this)
```

Comment:
Ah, my bad. For my own knowledge, what is the downside of this? Just that
one would have to continually bind it vs it being bound for every use?

On Fri, Nov 13, 2015 at 3:08 PM, Jimmy Jia <notifications@github.com> wrote:

> In modules/__tests__/Router-test.js
> <https://github.com/rackt/react-router/pull/2540#discussion_r44828136>:
>
> > @@ -272,7 +268,7 @@ describe('Router', function () {
> >          render() {
> >            const { label, children } = this.props
> >            const child = React.cloneElement(children, {
> > -            createElement: this.createElement
> > +            createElement: this.createElement.bind(this)
>
> Nit - please bind this in the constructor. Even though this is in test
> code, I'd rather not users accidentally follow this pattern without
> recognizing the consequences.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rackt/react-router/pull/2540/files#r44828136>.
>